pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -81,3 +89,83 @@ func TestPipelineRef(t *testing.T) {
 		})
 	}
 }
+
+func TestGetPipelineFunc(t *testing.T) {
+	// Set up a fake registry to push an image to.
+	s := httptest.NewServer(registry.New())
+	defer s.Close()
+	u, err := url.Parse(s.URL)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	testcases := []struct {
+		name            string
+		localPipelines  []runtime.Object
+		remotePipelines []runtime.Object
+		ref             *v1beta1.PipelineRef
+		expected        runtime.Object
+	}{
```

Comment:
Nit: you can save some horizontal space by putting both {{s on the same line here.